source/features/add-dashboard-label-links.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import {wrap} from '../libs/dom-utils';
+import observeEl from '../libs/simplified-element-observer';
+
+function linkifyLabels(activity: HTMLElement): void {
+	const labels = select.all('.IssueLabel', activity);
+	if (labels.length === 0) {
```

Comment:
This check isn't necessary, the loop just won't run if length is 0. I'd rather see

```ts
 	for (const label of select.all('.IssueLabel', activity)) {
```

Reply:
Mainly did this to avoid querying for the `repository` if it isn't needed.
Guess it's not to important, so I can also change it if you like that style better.